ipynb/tests/Frequency Invariance Test.ipynb

```diff
@@ -0,0 +1,791 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup"
```

Comment:
OK, I added a header text at the top but kept the info down here that I think is clearest next to the test class instantiation.